app/features-json/login_json_controller.rb

```diff
@@ -0,0 +1,46 @@
+require_relative "../shared/controller_base"
+require_relative "../services/user_service"
+require_relative "../services/dot_keys_variable_service"
+require_relative "json_controller"
+
+require "jwt"
+require "json"
+
+module FastlaneCI
+  # Controller responsible of handling the login process using JWT token.
+  class LoginJSONController < ControllerBase
+    include JSONController
+
+    HOME = "/api"
+
+    post "#{HOME}/login" do
```

Comment:
since this is the login controller, i do think that HOME = "/api/login"
we should constrain it to that namespace for now, expanding it if we need to in the future.